qa-niaid.planx-pla.net/portal/gitops.json

```diff
@@ -34,11 +34,7 @@
         "chartType": "image",
         "imageLink": "/custom/sponsors/gitops-sponsors/sars-cov2.png"
       }],
-      "homepageChartNodes": [],
-      "emailSignUpForm": {
```

Comment:
you could leave this in since we'll re-deploying it soon anyway 🤷‍♀️

Reply:
i was asked to get rid of the component by Mingfei so its not needed